### ISSUE TITLE
Cobertura (XML) report: Condition-coverage per line does not work

### ISSUE THREAD

**Issue by grichka** [istanbuljs/nyc#530]
Hello, 
I noticed that condition-coverage per line is not working. Is this a bug? is there an option that would not be documented to enable it?

## Expected Behavior
This is the cobertura (XML) report I get using istanbul in command line. 
`istanbul cover node_modules/mocha/bin/_mocha --report cobertura -- --require test/setup.js --reporter spec --check-leaks **/*.spec.js`
There is branch="true" and a condition-coverage attribute on line 23.
``` xml
  <lines>
        <line number="3"  hits="1"  branch="false" />
        <line number="10"  hits="1"  branch="false" />
        <line number="13"  hits="1"  branch="false" />
        <line number="15"  hits="1"  branch="false" />
        <line number="21"  hits="1"  branch="false" />
        <line number="22"  hits="0"  branch="false" />
        <line number="23"  hits="0"  branch="true"  condition-coverage="50% (1/2)" />
        <line number="24"  hits="0"  branch="false" />
```
## Observed Behavior
This is the cobertura (XML) report I get using `nyc --reporter cobertura npm test`
(where `npm test` is `mocha --require test/setup.js --check-leaks --reporter spec **/*.spec.js`.
There is branch="false" and a condition-coverage attribute is missing on line 23.
``` xml
<lines>
        <line number="3"  hits="1"  branch="false" />
        <line number="10"  hits="1"  branch="false" />
        <line number="13"  hits="1"  branch="false" />
        <line number="15"  hits="1"  branch="false" />
        <line number="21"  hits="1"  branch="false" />
        <line number="22"  hits="0"  branch="false" />
        <line number="23"  hits="0"  branch="false"/>
        <line number="24"  hits="0"  branch="false" />
```

## Forensic Information

**Operating System:** Microsoft Windows 10 and RHEL 6.5 (any)
**Environment Information:** https://gist.github.com/grichka/93cbcbb4813c4286c3e1967fe4257a60#file-output-txt

**Comment by mintel-ssallay** [comment]
I'm having the same issue with branch coverage at work, but we have a fix in place from before I started that seems to work.

It appears that [this line in istanbul-lib-coverage](https://github.com/istanbuljs/istanbuljs/blob/master/packages/istanbul-lib-coverage/lib/file.js#L207) is at fault. It's accessing a non-existent property on the members of the branchMap object. We manually replace that line using sed as part of our CI job.

Here's our working version:
```javascript
var line = branchMap[k].loc.start.line
```
I made a case on the istanbuljs monorepo (istanbuljs/istanbuljs#42) to try to get a fix in.

**Comment by bcoe** [comment]
@mintel-ssallay @grichka could you let me know if this is fixed with, `npm i nyc@10.3.0`?

**Comment by grichka** [comment]
Hi ! Thank you for your answers and fix !
I tested your fix few minutes ago and this new version seems to produce correct reports. 
I now get similar reports as I got with istanbul. 
The `condition-coverage` attribute is no more missing.

Great job
